src/main/webapp/request.jsp

```diff
@@ -4,7 +4,8 @@
         Request for an Account
     </h1>
     <div id="contentHolder">
-        <iframe src="https://spreadsheets.google.com/embeddedform?formkey=dDNsQmU4QXVYTVRhMjA2dEJWYW82Umc6MQ"
+	<p> Cannot see the request form below? Click <a href="https://docs.google.com/forms/d/e/1FAIpQLSfmiNsVnVANdB1-cOwkfn9l8Ts8eN-CtolLQwi93Nrug0sngw/viewform?embedded=true&formkey=dDNsQmU4QXVYTVRhMjA2dEJWYW82Umc6MQ" target="_blank">here.</a></p>
```

Comment:
Also, since you're using `target="_blank"`, you may want to add `rel="noopener noreferrer"` as per #6368.